cmd/eksctl-anywhere/cmd/installpackagecontroller.go

```diff
@@ -66,6 +66,8 @@ func installPackageController(ctx context.Context) error {
 		helmChart.Tag(),
 	)
 
+	fmt.Println("tag: " + helmChart.Tag() + " Image: " + helmChart.Image())
```

Comment:
Don't want to use logger here? I understand if we are trying to relay the output from kubectl but that's not what we are doing here.

Reply:
Whoops debug point lol. Taking it out.